__tests__/app.test.js

```diff
@@ -224,4 +224,36 @@ describe("express app", () => {
       });
     });
   });
+  describe.only('GET /api/articles', () => {
```

Comment:
```suggestion
  describe.only('GET /api/articles', () => {
```
Be careful of committing `.only`'s. It's a real easy way of pushing failing tests 😁